### DISCUSSION TITLE
Does anyone use Compiled with Rollup or Vite / Snowpack?

### DISCUSSION THREAD

**Discussion by pure-engineering** [atlassian-labs/compiled#1092] [General]
In SSR scenario, we can use
```javascript
import { CssCollector } from 'some-css-in-js-tool';
```
but it will not work in Dev mode

**Comment by miloofcroton** [comment]
I'm interested to know if there's a plan to support Vite in the future. This would mean it supports Remix as well as Next. I'm not convinced Remix is better than Next, but it would be nice if the use of Compiled wasn't contingent on also using Next.

**Comment by NikAtIdeally** [comment]
I got it working with Vite using non swc Vite (quite straight forward with the babel plugin). But I was not able to get the hot reload module to work properly, so trying to switch to Rsbuild now. It works a lot better, but I am not able to get jest to work with compiled now.
